tomcat/tomcat-jta/src/main/java/org/jboss/narayana/tomcat/jta/NarayanaJtaServletContextListener.java

```diff
@@ -64,6 +66,23 @@
     private static final List<String> DEFAULT_EXPIRY_SCANNERS = Arrays
             .asList(ExpiredTransactionStatusManagerScanner.class.getName());
 
+    /**
+     * Initialize and start Narayana JTA services.
+     * <p>
+     * During the setup node identifier, recovery modules, orphan filters, and expiry scanners are setup. Configuration file
```

Comment:
Is there a javadoc link we can use to say where this "Configuration" is?

{@link JTAEnvironmentBean} probably has most of what you are intending.